lib/engine/game/g_18_ireland/misc_config.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Engine
```

Comment:
do we really need two configs?

Reply:
it's probably worth the discussion with others, personally i like things split into categories so i can quickly find them. But having multiple files can be awkward as well.

Reply:
discussion complete